web/packages/teleterm/src/ui/ThemeProvider/theme.ts

```diff
@@ -15,155 +15,15 @@ limitations under the License.
-
-    surface: '#222C59',
-    surfaceSecondary: '#182047',
-
-    elevated: '#2D3761',
-
-    popout: '#4A5688',
-  },
-
-  brand: '#512FC9',
-
-  // Spot backgrounds are used as highlights, for example
-  // to indicate a hover or active state for an item in a menu.
-  spotBackground: [
-    'rgba(255,255,255,0.07)',
-    'rgba(255,255,255,0.13)',
-    'rgba(255,255,255,0.18)',
-  ],
-
-  // missing
-  inverse: '#B0BEC5',
-  progressBarColor: '#00BFA5',
-
-  dark: '#000000',
-
-  light: '#FFFFFF',
-
-  text: {
-    // The most important text.
-    primary: 'rgba(255,255,255,0.87)',
-    // Secondary text.
-    secondary: 'rgba(255, 255, 255, 0.56)',
-    // Placeholder text for forms.
-    placeholder: 'rgba(255, 255, 255, 0.24)',
-    // Disabled text have even lower visual prominence.
-    disabled: 'rgba(0, 0, 0, 0.24)',
-    // For maximum contrast.
-    contrast: '#FFFFFF',
-    // For text on  a background that is on a color opposite to the theme. For dark theme,
-    // this would mean text that is on a light background.
-    primaryInverse: '#000000',
-  },
-
-  buttons: {
-    text: 'rgba(255,255,255,0.87)',
-    textDisabled: 'rgba(255, 255, 255, 0.3)',
-    bgDisabled: 'rgba(255, 255, 255, 0.12)',
-
-    primary: {
-      text: '#FFFFFF',
-      default: '#512FC9',
-      hover: '#651FFF',
-      active: '#354AA4',
-    },
-
-    secondary: {
-      default: 'rgba(255,255,255,0.07)',
-      hover: 'rgba(255,255,255,0.13)',
-      active: 'rgba(255,255,255,0.18)',
-    },
-
-    border: {
-      default: 'rgba(255,255,255,0)',
-      hover: 'rgba(255, 255, 255, 0.07)',
-      active: 'rgba(255, 255, 255, 0.13)',
-      border: 'rgba(255, 255, 255, 0.36)',
-    },
-
-    warning: {
-      text: '#000000',
-      default: '#FF6257',
-      hover: '#FF8179',
-      active: '#FFA19A',
-    },
-
-    trashButton: {
-      default: 'rgba(255, 255, 255, 0.07)',
-      hover: 'rgba(255, 255, 255, 0.13)',
-    },
-
-    link: {
-      default: '#009EFF',
-      hover: '#33B1FF',
-      active: '#66C5FF',
-    },
-  },
-
-  grey: {
-    ...blueGrey,
-  },
-
-  error: {
-    main: '#FF6257',
-    hover: '#FF8179',
-    active: '#FFA19A',
-  },
-
-  warning: {
-    main: '#FFAB00',
-    hover: '#FFBC33',
-    active: '#FFCD66',
-  },
-
-  action: {
-    active: '#FFFFFF',
-    hover: 'rgba(255, 255, 255, 0.1)',
-    hoverOpacity: 0.1,
-    selected: 'rgba(255, 255, 255, 0.2)',
-    disabled: 'rgba(255, 255, 255, 0.3)',
-    disabledBackground: 'rgba(255, 255, 255, 0.12)',
-  },
-
-  subtle: '#CFD8DC',
-  link: '#039BE5',
-
-  danger: pink.A400,
-  highlight: yellow[50],
-  disabled: blueGrey[500],
-  info: lightBlue[600],
-  success: teal.A700,
-};
-
-const borders = [
-  0,
-  '1px solid',
-  '2px solid',
-  '4px solid',
-  '8px solid',
-  '16px solid',
-  '32px solid',
-];
+import { darkTheme } from "design/theme";
 
 const sansSerif = 'system-ui';
 
 const theme = {
   name: 'dark',
-  colors,
+  colors: darkTheme.colors,
```

Comment:
This is super nice. I did try replacing it with the light theme and it doesn't look 100% right, which suggests that some of our components make the wrong assumptions about the theme. But we can take care of these problems when the time comes to add the light theme to Connect.